### ISSUE TITLE
EEND-EDA Der's performance in libri2mix is much higher than EEND-SS's paper

### ISSUE THREAD

**Issue by Cerbera-odollam** [espnet/espnet#5632]
I have been conducting experiments using Libri2Mix, and I encountered an issue after stage 6.
I have a question regarding the performance of EEND-EDA, as I'm unable to reproduce the results. 
I'm curious about the reasons behind this.

result table : 
  diarized_test
  |threshold_median_collar|DER|
  |---|---|
  |result_th0.3_med1_collar0.0|28.10|
  |result_th0.4_med1_collar0.0|28.03|
  |result_th0.5_med1_collar0.0|28.06|
  |result_th0.6_med1_collar0.0|28.22|
  |result_th0.7_med1_collar0.0|28.61|

**Comment by sw005320** [comment]
Thanks for your report.
Which config file did you use?

**Comment by Cerbera-odollam** [comment]
i use train_diar_eda.yaml and decode_diar_eda.yaml

**Comment by sw005320** [comment]
Thanks.
@soumimaiti, can you answer it?

**Comment by soumimaiti** [comment]
@Cerbera-odollam Do you see training /validation DER as high too ?

**Comment by Cerbera-odollam** [comment]
no. when training/validation, DER is low enough. But when inference, DER gonna too high

**Comment by soumimaiti** [comment]
@Cerbera-odollam That seems like a mismatch. I can think of two reason: 

1) training mix is not matching to test? For example: If noise is augmented ? (This line should be uncommented for the first time) https://github.com/espnet/espnet/blob/master/egs2/librimix/enh_diar1/local/generate_librimix_sd.sh#L73 
2) Some error in inference. You can check devset inference DER for this.

**Comment by Cerbera-odollam** [comment]
After a careful analysis of the combined training and test mix datasets, it was observed that issues resembling those related to augmentation did not arise. Instead, it seems that concerns are exclusive to the inference stage, hinting at a potential problem within the supplementary inference code. Would it be possible for you to kindly investigate this matter further when time permits?

**Comment by soumimaiti** [comment]
@Cerbera-odollam What inference_config and command are you using?

**Comment by Cerbera-odollam** [comment]
you mean, config file? 
i use 
"${python} -m espnet2.bin.diar_inference \
                    --ngpu "${_ngpu}" \
                    --fs "${fs}" \
                    --data_path_and_name_and_type "${_data}/${_scp},speech,${_type}" \
                    --key_file "${_logdir}"/keys.JOB.scp \
                    --train_config "${diar_exp}"/config.yaml \
                    --model_file "${diar_exp}"/"${inference_model}" \
                    --output_dir "${_logdir}"/output.JOB \
                    ${_opts} || { cat $(grep -l -i error "${_logdir}"/diar_inference.*.log) ; exit 1; }"
